understanding/20/language-of-page.html

```diff
@@ -11,12 +11,12 @@ <h1>Understanding Language of Page</h1>
    <section id="brief">
 		<h2>In brief</h2>
 		<dl>
-		<dt>Objective</dt><dd>Assistive technology can determine the language of a page</dd>
-		<dt>Author task</dt><dd>Indicate the predominant language on a page</dd>
-		
-		</dl>
-		
-		</section>
+         <dt>Goal</dt><dd>Assistive technology can determine the language of a page.</dd>
+         <dt>What to do</dt><dd>Indicate the predominant language on a page.</dd>
+         <dt>Why it's important</dt><dd>People using assistive technology get information in the correct language.</dd> 
```

Comment:
@GreggVan suggested
```suggestion
         <dt>Why it's important</dt><dd>People using assistive technology get information in the correct language and synthesizers will pronounce text properly.</dd> 
```

Reply:
I do not believe either of these suggested additions make this briefer or adds meaning to anyone newly encountering the concept, who would be unlikely to know what a speech synthesizer is. (The term is also not used in the Understanding document).

I believe it is better as it is.